### PR TITLE
Task list output verbosity

### PR DESCRIPTION
**Pull request by User3574**
Closes: #426

### CODE REVIEW

**Comment by Kobzol** [comment]
For Unix tools it's common to select verbosity with short flags (`-v`, `-vv`, `-vvv`). I tried to look at how it can be done using `clap`, but it was a bit complicated, so I took the implementation from https://docs.rs/clap-verbosity-flag/latest/clap_verbosity_flag/ and implemented it here.

The default should be brief verbosity, otherwise it's not very useful, and it couldn't be disabled anyway with the short flags.

Could you please fix tests? They will need to use the verbose mode. Thanks.

**Review by Kobzol — CHANGES_REQUESTED** [review]

**Comment by Kobzol** [comment]
When the error is hidden, it might be confusing to the user, since we hide important information from him. Maybe it would be better if we always shown the error column, but if the output mode is not verbose, we truncate it. I suggest to truncate the error if it's longer than `X` (16) characters or if it contains a newline.

Then we should show some message that says "Use -v to display full error.", and place it either below the table or in the error column. @spirali thoughts?

**Comment by spirali** [comment]
> When the error is hidden, it might be confusing to the user, since we hide important information from him. Maybe it would be better if we always shown the error column, but if the output mode is not verbose, we truncate it. I suggest to truncate the error if it's longer than `X` (16) characters or if it contains a newline.
> 
> Then we should show some message that says "Use -v to display full error.", and place it either below the table or in the error column. @spirali thoughts?

It seems like a good idea: hame some small fragment by default and full info by "-v".

File: crates/hyperqueue/src/client/output/cli.rs [hunk 1]
```diff
@@ -632,6 +644,7 @@ impl Output for CliOutput {
         ]);
 
         self.print_horizontal_table(rows, header);
+        log::info!("Use -v to display full error.");
```
Comment (Kobzol) [inline crates/hyperqueue/src/client/output/cli.rs:647]:
```suggestion
        log::info!("An error message was truncated. Use -v to display the full error.");
```
+ only print this if at least one error was actually truncated.

File: crates/hyperqueue/src/client/output/cli.rs [hunk 2]
```diff
@@ -603,8 +607,16 @@ impl Output for CliOutput {
...
                                 ("Stderr", stderr),
                             ]),
-                            match &task.state {
-                                JobTaskState::Failed { error, .. } => {
+                            match (verbosity, &task.state) {
+                                (Verbosity::Normal, JobTaskState::Failed { error, .. }) => {
+                                    let mut error_mut = error.clone();
+                                    if error_mut.len() >= ERROR_TRUNCATE_LENGTH {
+                                        error_mut.truncate(ERROR_TRUNCATE_LENGTH);
+                                        error_mut = error_mut.add("...");
```
Comment (Kobzol) [inline crates/hyperqueue/src/client/output/cli.rs:615]:
```suggestion
                                        error_mut.push_str("...");
```

**Review by Kobzol — CHANGES_REQUESTED** [review]

**Comment by Kobzol** [comment]
Great! :) Please rebase on master & squash commits and I'll merge it.

**Review by Kobzol — APPROVED** [review]

**Pull request merged**